### DISCUSSION TITLE
Reforme de la sécurité sociales pour les cotisations 2025

### DISCUSSION THREAD

**Discussion by NicolasFGT** [betagouv/mon-entreprise#3113] [Q&A] [answered]
Bonjour,

Avez-vous déjà commencé à travailler sur la reforme de la sécurité sociales pour les cotisations 2025, avec le changement du calcul des assiettes de cotisations ?

Cordialement,
Nicolas FERNANDES

**Comment by liliced** [comment, marked as answer]
Bonjour,

Nous sommes désolés pour le délai de notre réponse. Comme vous l'avez peut-être remarqué, la réforme est désormais implémentée dans les simulateurs pour indépendant⋅es (mais pas dans le comparateur de statuts).
